content/es/docs/reference/glossary/volume.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Volume
+id: volume
+date: 2018-04-12
+full_link: /docs/concepts/storage/volumes/
+short_description: >
+  Un directorio que contiene datos y que es accesible desde los contenedores corriendo en un pod.
+
+aka: 
+tags:
+- core-object
+- fundamental
+---
+ Un directorio que contiene datos y que es accesible desde los contenedores corriendo en un {{< glossary_tooltip text="pod" term_id="pod" >}}.
+
+<!--more--> 
+
+Un volumen de Kubernetes viven mientras el {{< glossary_tooltip text="pod" term_id="pod" >}} esta corriendo. Un volumen persiste mientra exista el {{< glossary_tooltip text="pod" term_id="pod" >}}, no depende de la vida del  {{< glossary_tooltip text="container" term_id="container" >}}, y por eso se conservan los datos entre los reinicios de los mismos. 
```

Comment:
```suggestion
Un volumen de Kubernetes viven mientras el {{< glossary_tooltip text="pod" term_id="pod" >}} esta corriendo. Un volumen persiste mientra exista el {{< glossary_tooltip text="pod" term_id="pod" >}}, no depende de la vida del  {{< glossary_tooltip text="contenador" term_id="container" >}}, y por eso se conservan los datos entre los reinicios de los mismos. 
```
?

Reply:
Hi @sftim , why did you change "container" for "contenador"?

Reply:
I think he meant 'contenedor'.

Reply:
It makes sense.

Reply:
Oops, contenedor indeed.